hs_core/views/resource_folder_hierarchy.py

```diff
@@ -27,21 +27,21 @@ def data_store_structure(request):
     """
     res_id = request.POST.get('res_id', None)
     if res_id is None:
-        return HttpResponse(content_type='application/json', status=400)
+        return HttpResponse('Bad request - resource id is not included', status=400)

```

Comment:
@hyi I am sure there is no rule against setting a specific content-type or returned data format in http response. What I am suggesting is that we use a consistent data format (which is json) for http response for responding to ajax call. I am OK if you not make this change.